packages/lesswrong/components/tagging/TagPage.tsx

```diff
@@ -1393,6 +1446,11 @@ const TagPage = () => {
             Next Tag ({nextTag.name})
         </Link></span>}
       </span>}
+      {revision && <Link to={tagGetUrl(tag, {lens: selectedLens?.slug})} className={classes.revisionNotice}>
```

Comment:
is this with the fixed functionality for returning to the same lens?

Reply:
I don't understand the question. This is you're viewing a revision of a page, it's a link to take you to the latest version of that page